src/components/Popups/PopupItem.tsx

```diff
@@ -1,40 +1,49 @@
 import { useEffect } from 'react'
 import { X } from 'react-feather'
-import styled, { useTheme } from 'styled-components/macro'
+import styled, { css, useTheme } from 'styled-components/macro'
 
 import { useRemovePopup } from '../../state/application/hooks'
 import { PopupContent } from '../../state/application/reducer'
 import FailedNetworkSwitchPopup from './FailedNetworkSwitchPopup'
 import TransactionPopup from './TransactionPopup'
 
-const StyledClose = styled(X)<{ isTransactionPopup: boolean }>`
+const StyledClose = styled(X)<{ padding: number }>`
   position: absolute;
-  right: ${({ isTransactionPopup }) => (isTransactionPopup ? '16px' : '20px')};
-  top: ${({ isTransactionPopup }) => (isTransactionPopup ? '16px' : '20px')};
+  right: ${({ padding }) => `${padding}px`};
+  top: ${({ padding }) => `${padding}px`};
 
   :hover {
     cursor: pointer;
   }
 `
-const Popup = styled.div<{ isTransactionPopup: boolean; show: boolean }>`
+const Popup = css<{ show: boolean }>`
```

Comment:
This isn't a class/component, so change it to `popupCss`

Reply:
done